packages/ember-metal/lib/index.js

```diff
@@ -379,6 +379,16 @@ Object.defineProperty(Ember, 'ENV', {
 });
 
 /**
+ * The context that Ember searches for namespace instances on.
+ * @private
```

Comment:
Looks like the JSCS rule that requires `@private` / `@public` doesn't like it when the comments include a leading `* ` like this.